packages/styled-components/src/utils/hasThemeReferences.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import { Parser } from 'acorn';
```

Comment:
There has to be a lighter way of doing this, like calling `.toString()` on the interpolation function and looking for `.theme`. We definitely won't want to pull in all of acorn for development